android/app/build.gradle

```diff
@@ -137,6 +137,8 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-vector-icons')
+    compile project(':react-native-vector-icons')
```

Comment:
Looks like duplicate line

Reply:
Wait I don't remember changing these or any of the other Android and iOS files...